File: C/PRAK304-2210817110006-MohammadZakiFirmansah.c
#include <stdio.h>
int main(){
    int nilai;
    printf("Masukkan Nilai = "); 
    scanf("%d", &nilai);
    if(nilai>=100){
        printf("Anda Menginput Melebihi Limit Bilangan");
    }else if(nilai>1 && nilai<10){
        printf("Satuan");
    }else if (nilai>9 && nilai<20){
        printf("Belasan");
    }else if(nilai>=20 && nilai<100){
        printf("Puluhan");
    }else {
        printf("Nol");
    }
}

File: Python/PRAK304-2210817110006-MohammadZakiFirmansah.py
a = int(input("Masukkan Nilai : "))
if(a >= 100) :
     print("Anda Menginput Melebihi Limit Bilangan")
elif(a > 1 and a < 10) :
    print("Satuan")
elif (a > 9 and a < 20) :
    print("Belasan")
elif(a >= 20 and a < 100) :
    print("Puluhan")
else :
    print("Nol")